### PR TITLE
Rent strike colors

### PR DESCRIPTION
**Pull request by river-honer**
* Updates the colours for the cluster marker and the icons for rent strikes.
* Use SVG icon
* Update text in sidebar
* Put orange color on text in sidebar, as a kind of basic legend.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -112,13 +112,10 @@ a:hover {
   color: #d01c8b;
 }
 
-.color-ocurring {
-  color: #1772aa;
+.color-rent-strike {
```
[the remark this answers is not in the document]
Comment (clhenrick) [reply styles.css:115]: @river-honer we could also stash this color as a CSS variable at the top:

```css
:root {
  --rent-strike-color: #DB3A1D;
}

.color-rent-strike {
  color: var(--rent-strike-color);
}

.marker-cluster, .marker-cluster div {
  background-color: var(--rent-strike-color);
}
```

File: styles.css [hunk 2]
```diff
@@ -71,10 +71,10 @@ details p:last-of-type {
   margin-bottom: 0;
 }
 
-/* marker cluster override */
-.marker-cluster-small div {
-  background-color: #ff8c1a !important;
-  opacity: 0.6;
+.marker-cluster, .marker-cluster div { 
+  background-color: #FF6347;
```
Comment (clhenrick) [inline styles.css:75]: use CSS color variable here (see other comment on `.color-rent-strike`)

File: assets/mapIcons/rent-strike.svg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<svg width="40" height="40" xmlns="http://www.w3.org/2000/svg"><g fill="#FF6347" fill-rule="nonzero" stroke="#FFF"><path d="M10.853 23.982l-3.354-9.388h6.226L9.14 1.766H2.871l3.355 9.388H0l10.853 12.828zM31.309 9.961L26.76 6.613V2.621c0-.625-.508-1.133-1.134-1.133h-1.84c-.625 0-1.133.508-1.133 1.133v.971l-3.468-2.555c-.753-.553-1.984-.553-2.739 0l-5.123 3.771L12.67 8.74l3.776-2.779c.755-.555 1.986-.555 2.739 0l9.774 7.193.335-.457a1.701 1.701 0 01-2.375 0l4.39-2.736z"/><path d="M27.69 13.781l-8.501-6.283v.004l-.007-.004c-.753-.555-1.98-.555-2.733 0l-3.355 2.48 1.842 5.377H8.803l2.769 9.873-4.988-4.877v4.346c0 1.48 1.269 2.68 2.831 2.68h5.193v-8.055c0-.625.509-1.135 1.132-1.135h4.153c.623 0 1.132.51 1.132 1.135v8.055h5.201c1.563 0 2.832-1.199 2.832-2.678v-9.112c-.001-.437-.617-1.249-1.368-1.806h0z"/></g></svg>
```
Comment (clhenrick) [inline assets/mapIcons/rent-strike.svg:1]: change `fill` color in the `g` element to `#FF6347`
(see my comment on the CSS class `.color-rent-strike`)

**Comment by river-honer** [comment]
Excellent, will make these changes. I prefer the colour you have chosen. Thanks for the review!

File: styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (river-honer) [reply styles.css:115]: Done

File: styles.css (hunk 2 above)
Comment (river-honer) [reply styles.css:75]: Great, looking forward to uDone

File: assets/mapIcons/rent-strike.svg (hunk 3 above)
Comment (river-honer) [reply assets/mapIcons/rent-strike.svg:1]: Done

**Comment by clhenrick** [comment]
@river-honer thanks! One last ask: would you mind removing the old PNG icons? We no longer need to hold onto them.

**Comment by clhenrick** [comment]
Okay, added a 1px white border to the clusters, think it's good to merge.

**Pull request merged**